### DISCUSSION TITLE
Word Count for New Testament?

### DISCUSSION THREAD

**Discussion by andrewyu47** [openenglishbible/Open-English-Bible#352] [Q&A]
Hi!

Trying to calculate: what's the word count for the NET bible version?
In particular, what's the word count of the New Testament for NET?

**Comment by openenglishbible** [comment]
I can't help you with the NET version, which isn't our translation.

But for the OEB I suspect the easiest way would be to download the Microsoft Word version, open in a word processor such as Word or LibreOffice, delete the OT and use the Word processor's inbuilt word count function.

Alternatively, you could start with the plain text version.

**Comment by andrewyu47** [comment]
Ah thank you!  I was looking at multiple versions and got confused lol.

Totally don't know why I didn't think of that.
